tasks/go.py

```diff
@@ -174,16 +174,31 @@ def misspell(ctx, targets):
         print("misspell found no issues")
 
 @task
-def deps(ctx):
+def deps(ctx, no_checks=False, core_dir=None, verbose=False):
```

Comment:
Can you document how a developer is supposed to use these options when setting up their dev environment? Could be documented in the command help, might be worth updating the `docs/dev/` if applicable.